src/components/Menu/MenuItemLink.tsx

```diff
@@ -0,0 +1,140 @@
+import * as _ from 'lodash'
+import * as React from 'react'
+import * as PropTypes from 'prop-types'
+
+import { createShorthandFactory, customPropTypes, UIComponent } from '../../lib'
+import { Extendable } from '../../../types/utils'
+import Icon from '../Icon/Icon'
+import { MenuItemBehavior } from '../../lib/accessibility'
+import { Accessibility, AccessibilityActionHandlers } from '../../lib/accessibility/interfaces'
+
+class MenuItemLink extends UIComponent<Extendable<{}>, {}> {
+  static displayName = 'MenuItemLink'
+
+  static className = 'ui-menu__itemlink'
+
+  static create: Function
+
+  static propTypes = {
+    /** A menu item can be active. */
+    active: PropTypes.bool,
+
+    /** An element type to render as (string or function). */
+    as: customPropTypes.as,
+
+    /** Primary content. */
```

Comment:
Please add `@docSiteIgnore` to the comment so that it won't show in the doc site approved props for users.